packages/ui/src/Avatar/index.tsx

```diff
@@ -6,12 +6,14 @@ import common from '../styles/common.module.css';
 import {ISize} from '@eruditorgroup/profi-toolkit';
 
 export interface AvatarProps
-  extends Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
+  extends Omit<React.HTMLAttributes<HTMLImageElement>, 'children'> {
```

Comment:
если переопределили поведелние loading,  то нужно убрать из типа
```suggestion
  extends Omit<React.HTMLAttributes<HTMLImageElement>, 'children' | 'loading'> {
```